profiles/base-empty.yaml

```diff
@@ -67,3 +67,4 @@ empty_values:
   held_athletics_items: []
   training_list: []
   stabbity: {}
+  telescope_storage: []
```

Comment:
❓ Shouldn't this be a hash? `telescope_storage: {}`

Reply:
Yes